Classes/Eel/ZoomApiHelper.php

```diff
@@ -18,28 +21,43 @@ class ZoomApiHelper implements ProtectedContextAwareInterface {
      */
     protected $zoomApiService;
 
+    /**
+     * @Flow\Inject
+     * @var LoggerInterface
+     */
+    protected $systemLogger;
+
     /**
      * See also https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/recordingget
-     * 
+     *
      * @param DateTime|string $from
      * @param DateTime|string $to
-     * @return array
-     * @throws Exception
+     *
+     * @return array|false
```

Comment:
It would be nice to describe here in which case `false` can be returned (then this is also clear at first glance in the quick function definition).